en/dev_log/log_encryption.md

```diff
@@ -160,61 +190,53 @@ You can now build and test.
 ## Download & Decrypt Log Files
 
 Encrypted log files are downloaded using the QGroundControl [Log Download](https://docs.qgroundcontrol.com/master/en/qgc-user-guide/analyze_view/log_download.html) view (**Analyze Tools > Log Download**) just like ordinary log files.
-The only difference is that for each flight you will need to download both the encrypted log file, and the file containing the encrypted symmetric key.
 
-The encrypted log file and encrypted symmetric key file are displayed with a timestamp (but no filename) in QGroundControl, as shown below.
-You can determine which files are associated based on their timestamps.
+Note that the encrypted files will be downloaded with the `.ulg` suffix, instead of `.ulge`.
 
-![QGroundControl ULog Download](../../assets/qgc/analyze/encrypted_log.png)
+### Decrypt ULogs
 
-Select and download both files.
+Before you can analyze your encrypted logs, you will need to decrypt them.
+There is a Python script that can be used to decrypt logs in `Tools/decrypt_ulog.py`.
 
-Note that both files will be downloaded with the `.ulg` suffix.
-You can identify the symmetric key file, as it is usually much smaller than the log file (about 300 bytes)
+When decrypting a `.uloge` file the script takes 3 arguments:
```

Comment:
FYI, updated instructions to just be how you do it for uloge - makes things a bit simpler, which is the benefit of this whole approach. 

I kept the command line instructions, but pushed them below.

Reply:
Just minor comment, the encrypted file extension is .ulge, without the 'o' in the middle.

Reply:
I fixed that